bin/boot

```diff
@@ -26,49 +26,53 @@ function login  {
     set -e
 }
 
-function _update_odoo_conf_param {
-    if [ "$2" ]; then
-        echo $log_src[`date +%F.%H:%M:%S`]' Replacing parameter '$1
-        sudo -i -u \#$TARGET_ID sed -i "s/$1/$2/g" $odoo_conf_file
-    fi
+function _update_odoo_conf_params {
+    # Loop over all the "ODOO_" ENV variables
+    while read -r var; do
+        # Remove "ODOO_" from ENV variable and convert to lowercase
+        param=${var:5}
+        param=${param,,}
+
+        # Get the value of the corresponding ENV variable and escape slashes
+        val=${!var}
+        val=$( echo $val | sed 's/\//\\\//g')
+
+        bash /app/bin/update_odoo_param.sh $odoo_user $odoo_conf_file $param $val
+
+        # Unset the environment variable for security purpose
+        unset $var
+    done <<< "$( printenv | grep '^ODOO_' | sed 's/=.*//g' )"
 }
 
-function start {
-    echo $log_src[`date +%F.%H:%M:%S`]' Checking special requirements...'
-    bash /opt/scripts/startup.sh
-
-    echo $log_src[`date +%F.%H:%M:%S`]' Updating Odoo conf...'
-    _update_odoo_conf_param '_ADMIN_PASSWORD_' $ADMIN_PASSWORD
-    _update_odoo_conf_param '_DB_USER_' $DB_USER
-    _update_odoo_conf_param '_DB_PASSWORD_' $DB_PASSWORD
-    _update_odoo_conf_param '_DB_FILTER_' $DB_FILTER
-    echo $log_src[`date +%F.%H:%M:%S`]' Update Odoo conf done'
+function _setup_ssh_key {
+    echo $log_src[`date +%F.%H:%M:%S`]' Setting up SSH key...'
+    ssh_folder=$( getent passwd $odoo_user | cut -d: -f6 )/.ssh
+    echo $log_src[`date +%F.%H:%M:%S`]' Creating SSH folder = '$ssh_folder
+    mkdir $ssh_folder
+    echo $log_src[`date +%F.%H:%M:%S`]' Copy SSH private key from /opt/odoo/ssh'
+    cp /opt/odoo/ssh/id_rsa $ssh_folder
+    echo $log_src[`date +%F.%H:%M:%S`]' Scanning GitHub key...'
+    ssh-keyscan github.com > $ssh_folder/known_hosts
```

Comment:
> The file is empty since I just created the folder.

hehe: no. Maybe the folder existed before.

Not big deal because this is in a controlled environment but fixing the issue now should not be difficult (a TODO is fine too if unexpected complexity)